tests/android-test.ts

```diff
@@ -0,0 +1,146 @@
+jest.mock("danger", () => jest.fn());
+import danger from "danger";
+const dm = danger as any;
+
+import androidChecks from "../org/pr/android";
+
+// The mocked data and return values for calls the rule makes.
+async function getMockedDiff(customDiffs : string) {
+    let diffString : string = "diff --git a/WordPress/src/main/res/values/strings.xml a/WordPress/src/main/res/values/strings.xml\n";
+    diffString += "index eb1a3f999dc..87ac8c3793c 100644\n";
+    diffString += "--- a/WordPress/src/main/res/values/strings.xml\n";
+    diffString += "+++ b/WordPress/src/main/res/values/strings.xml\n";
+    diffString += "<string name=\"login_epilogue_screen_title\">Logged in as</string>\n";
+    diffString += customDiffs;
+
+    return diffString;
+}
+
+beforeEach(() => {
+    dm.warn = jest.fn().mockReturnValue(true);
+});
+
+describe("string checks", () => {
+    it("Warn when an entry which references another string resources is added without the translatable=false attribute", async () => {
```

Comment:
nit: the `it` syntax used in most test frameworks like here are named that way because they are meant to be read as English sentences 😉 

```suggestion
    it("warns when an entry which references another string resources is added without the translatable=false attribute", async () => {
```

Reply:
TIL! Thank you!